keywords.json

```
@@ -136,6 +136,7 @@
   "mortar-board": ["education", "learn", "teach"],
   "mute": ["quiet", "sound", "audio", "turn", "off"],
   "no-newline": ["return"],
+  "north-star": ["star", "snowflake", "asterisk"]
```

This will probably fix the failing build.

```suggestion
  "north-star": ["star", "snowflake", "asterisk"],
```